rapids/rapids.sh

```diff
@@ -105,33 +68,77 @@ readonly XGBOOST_GPU_SUB_VERSION=$(get_metadata_attribute 'spark-gpu-sub-version
 readonly SCALA_VER="2.12"
 
 # Dask config
+readonly DASK_RUNTIME="$(/usr/share/google/get_metadata_value attributes/dask-runtime || echo 'standalone')"
 readonly DASK_LAUNCHER=/usr/local/bin/dask-launcher.sh
 readonly DASK_SERVICE=dask-cluster
+readonly DASK_WORKER_SERVICE=dask-worker
+readonly DASK_SCHEDULER_SERVICE=dask-scheduler
 readonly DASK_YARN_CONFIG_FILE=/etc/dask/config.yaml
 
 # Dataproc configurations
 readonly SPARK_CONF_DIR='/etc/spark/conf'
 
 function execute_with_retries() {
-  local -r cmd=$1
-  for ((i = 0; i < 10; i++)); do
+  local -r cmd="$*"
+  for i in {0..9} ; do
     if eval "$cmd"; then
-      return 0
-    fi
+      return 0 ; fi
     sleep 5
   done
+  echo "Cmd '${cmd}' failed."
   return 1
 }
 
+readonly conda_env="/opt/conda/miniconda3/envs/dask-rapids"
 function install_dask_rapids() {
-  if is_debian11 || is_debian12 || is_ubuntu20 || is_ubuntu22 ; then
-      local python_ver="3.10"
-  else
-      local python_ver="3.9"
+  if is_cuda12 ; then
+    local python_spec="python>=3.10"
+    local cuda_spec="cuda-version>=12,<12.6"
+    local dask_spec="dask>=2024.5"
+    local numba_spec="numba"
+  elif is_cuda11 ; then
+    local python_spec="python>=3.9"
+    local cuda_spec="cuda-version>=11,<=11.8"
```

Comment:
Would it be alright to update this like so?

```suggestion
    local cuda_spec="cuda-version>=11,<12"
```

Prior discussion in this thread: https://github.com/GoogleCloudDataproc/initialization-actions/pull/1219#discussion_r1717406318